
> web@1.0.0 build C:\Users\med_i\OneDrive\Bureau\turbo\apps\web
> next build

   ▲ Next.js 14.0.4

   Creating an optimized production build ...
 ✓ Compiled successfully
   Linting and checking validity of types ...
 ⨯ ESLint: .eslintrc.js: 	Configuration for rule "@typescript-eslint/no-unsafe-assignment" is invalid: 	Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"false"'). 
   Collecting page data ...
   Generating static pages (0/5) ...
   Generating static pages (1/5) 
   Generating static pages (2/5) 
PrismaClientInitializationError: 
Invalid `prisma.ticket.findFirst()` invocation:

   Generating static pages (3/5) 
 ✓ Generating static pages (5/5) 

error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:3
   | 
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
   | 

Validation Error Count: 1
    at si.handleRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:7009)
    ... 7 lines matching cause stack trace ...
    at async r9 (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\app\page.js:50:82084) {
  code: 'INTERNAL_SERVER_ERROR',
  name: 'TRPCError',
  [cause]: PrismaClientInitializationError: 
  Invalid `prisma.ticket.findFirst()` invocation:
  
  
  error: Environment variable not found: DATABASE_URL.
    -->  schema.prisma:3
     | 
   2 |   provider = "postgresql"
   3 |   url      = env("DATABASE_URL")
     | 
  
  Validation Error Count: 1
      at si.handleRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:7009)
      at si.handleAndLogRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:6059)
      at si.request (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:5767)
      at async l (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:130:9661)
      at async C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:131:4992
      at async y.middlewares (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:9011)
      at async r (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8416)
      at async t (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8730)
      at async r9 (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\app\page.js:50:82084) {
    clientVersion: '5.8.0',
    errorCode: undefined
  }
}
PrismaClientInitializationError: 
Invalid `prisma.ticket.findFirst()` invocation:


error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:3
   | 
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
   | 

Validation Error Count: 1
    at si.handleRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:7009)
    ... 7 lines matching cause stack trace ...
    at async r9 (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\app\page.js:50:82084) {
  code: 'INTERNAL_SERVER_ERROR',
  name: 'TRPCError',
  [cause]: PrismaClientInitializationError: 
  Invalid `prisma.ticket.findFirst()` invocation:
  
  
  error: Environment variable not found: DATABASE_URL.
    -->  schema.prisma:3
     | 
   2 |   provider = "postgresql"
   3 |   url      = env("DATABASE_URL")
     | 
  
  Validation Error Count: 1
      at si.handleRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:7009)
      at si.handleAndLogRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:6059)
      at si.request (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:5767)
      at async l (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:130:9661)
      at async C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:131:4992
      at async y.middlewares (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:9011)
      at async r (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8416)
      at async t (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8730)
      at async r9 (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\app\page.js:50:82084) {
    clientVersion: '5.8.0',
    errorCode: undefined
  }
}

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
PrismaClientInitializationError: 
Invalid `prisma.ticket.findFirst()` invocation:


error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:3
   | 
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
   | 

Validation Error Count: 1
    at si.handleRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:7009)
    at si.handleAndLogRequestError (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:6059)
    at si.request (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:125:5767)
    at async l (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:130:9661)
    at async C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\940.js:131:4992
    at async y.middlewares (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:9011)
    at async r (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8416)
    at async t (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\chunks\494.js:1:8730)
    at async r9 (C:\Users\med_i\OneDrive\Bureau\turbo\apps\web\.next\server\app\page.js:50:82084)

> Export encountered errors on following paths:
	/page: /
 ELIFECYCLE  Command failed with exit code 1.
